bikesharing/migrations/0021_location_internal_location.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2020-03-20 11:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bikesharing', '0020_locationtracker_internal_only'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='location',
+            name='internal_location',
```

Comment:
this is a field on the `location` model. we should not duplicate the model name in the field again, so lets use just `internal`? :)

Reply:
renamed in 31b1b9d